tests/baselines/reference/generatorTypeCheck55.errors.txt

```diff
@@ -1,9 +1,12 @@
-tests/cases/conformance/es6/yieldExpressions/generatorTypeCheck55.ts(2,19): error TS9003: 'class' expressions are not currently supported.
+tests/cases/conformance/es6/yieldExpressions/generatorTypeCheck55.ts(2,29): error TS2507: Type 'any' is not a constructor function type.
+tests/cases/conformance/es6/yieldExpressions/generatorTypeCheck55.ts(2,30): error TS1163: A 'yield' expression is only allowed in a generator body.
 
 
-==== tests/cases/conformance/es6/yieldExpressions/generatorTypeCheck55.ts (1 errors) ====
+==== tests/cases/conformance/es6/yieldExpressions/generatorTypeCheck55.ts (2 errors) ====
     function* g() {
         var x = class C extends (yield) {};
-                      ~
-!!! error TS9003: 'class' expressions are not currently supported.
+                                ~~~~~~~
+!!! error TS2507: Type 'any' is not a constructor function type.
+                                 ~~~~~
+!!! error TS1163: A 'yield' expression is only allowed in a generator body.

```

Comment:
This is in a generator body, and I'm not seeing *Yield* getting turned off in this grammar:

> <span id="sec-14.5" class="secnum">[14.5](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-class-definitions "link to this section")</span> Class Definitions
======================================================================================================================================================================

> Syntax
------

> *ClassDeclaration*<sub>\[Yield, Default\]</sub> <span class="geq">:</span>

> &nbsp;&nbsp;&nbsp;`class` *BindingIdentifier*<sub>\[?Yield\]</sub> *ClassTail*<sub>\[?Yield\]</sub>

> &nbsp;&nbsp;&nbsp;<span class="grhsannot">\[+Default\]</span> `class` *ClassTail*<sub>\[?Yield\]</sub>

> *ClassExpression*<sub>\[Yield\]</sub> <span class="geq">:</span>

> &nbsp;&nbsp;&nbsp;`class` *BindingIdentifier*<sub>\[?Yield\]opt</sub> *ClassTail*<sub>\[?Yield\]</sub>

> *ClassTail*<sub>\[Yield\]</sub> <span class="geq">:</span>

> &nbsp;&nbsp;&nbsp;*ClassHeritage*<sub>\[?Yield\]opt</sub> `{` *ClassBody*<sub>\[?Yield\]opt</sub> `}`

> *ClassHeritage*<sub>\[Yield\]</sub> <span class="geq">:</span>

> &nbsp;&nbsp;&nbsp;`extends` *LeftHandSideExpression*<sub>\[?Yield\]</sub>